flexget/plugins/output/exec.py

```diff
@@ -104,6 +105,12 @@ def prepare_config(self, config):
             config = {'on_output': {'for_accepted': config}}
         if not config.get('encoding'):
```

Comment:
Something is wrong here, needs to test if config is an array and deal with it accordingly before.